scipy/special/tests/test_pdtr.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+from numpy import isnan
+
+import pytest
+from numpy.testing import (assert_almost_equal, assert_array_equal)
```

Comment:
Leave out the parens since this only spans one line.

Reply:
done